app/screens/convert_gm_to_channel/team_selector_list/index.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo, useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+
+import {preventDoubleTap} from '@app/utils/tap';
+import SearchBar from '@components/search';
+import TeamList from '@components/team_list';
+import {useTheme} from '@context/theme';
+import {popTopScreen} from '@screens/navigation';
+import {changeOpacity, getKeyboardAppearanceFromTheme} from '@utils/theme';
+
+const styles = StyleSheet.create({
+    container: {
+        padding: 12,
+    },
+    listContainer: {
+        marginTop: 12,
+    },
+});
+
+type Props = {
+    teams: Team[];
+    selectTeam: (teamId: string) => void;
+}
+
+const TeamSelectorList = ({teams, selectTeam}: Props) => {
+    const theme = useTheme();
+
+    const color = useMemo(() => changeOpacity(theme.centerChannelColor, 0.72), [theme]);
+
+    const [filteredTeams, setFilteredTeam] = useState(teams);
```

Comment:
I like `useState` before `useMemo`, but in this particular case I am 0/5.

Reply:
Done.